src/core/Class.js

```diff
@@ -28,7 +28,7 @@ Class.extend = function (props) {
 		this.callInitHooks();
 	};
 
-	const parentProto = NewClass.__super__ = this.prototype;

```

Comment:
That's `__super__`, not `__proto__`!

Reply:
Looks like I didn't have my morning coffee hahah